scripts/test.sh

```diff
@@ -53,6 +65,20 @@ else
   echo "***** Print conda info *****"
   conda info
   conda list
+  conda clean --yes --index-cache
```

Comment:
@hmaarrfk, this was just a debugging step. We should not release a miniforge with this requirement. This appears to be a bug in micromamba/mamba

Reply:
ah i misunderstood. I htink that @jakirkham and @ocefpaf might have gotten to the bottom of it.

Reply:
https://github.com/conda-forge/miniforge/pull/360